src/server/config/routes.js

```diff
@@ -29,6 +30,7 @@ module.exports = function (app) {
 
   // Get user GitHub Subscriptions
   app.get('/api/subscriptions/', users.userSub);
+  app.post('/api/repos/' , github.handlePost);
```

Comment:
our style convention would have the /api/repos request handler as repos.handlePost. so we should move query/github.js to request-handlers/repos.js